src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.LocalFunctions.cs

```diff
@@ -163,9 +165,14 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
             {
                 this.State = pending.State;
                 BoundNode branch = pending.Branch;
-                LeaveParameters(localFuncSymbol.Parameters, branch?.Syntax,
-                                branch?.WasCompilerGenerated == true
-                                    ? location : null);
+
+                if (branch != null)//No need to check the parameters if we dont know the branch
```

Comment:
Please place comments like this above the `if` block.  Our style doesn't put comments between the if conditional and the opening `{`